ksql-functional-tests/src/test/resources/query-validation-tests/elements.json

```diff
@@ -33,19 +24,29 @@
           "format": "AVRO"
         },
         {
-          "name": "S",
+          "name": "OUTPUT",
           "format": "AVRO",
           "partitions": 4
         }
       ],
       "inputs": [{"topic": "input", "value": {"c1": 4}}],
-      "outputs": [{"topic": "S", "value": {"C1": 4}}]
+      "outputs": [{"topic": "OUTPUT", "value": {"C1": 4}}]
     },
     {
-      "name": "validate AVRO ok with elements",
+      "name": "validate with elements OK",
+      "format": ["JSON"],
```

Comment:
did you mean to include `KAFKA` here as well?

Reply:
No, as this is covered by the all the tests to check Kafka format can handle a specific type of element.